src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -130,6 +132,7 @@ protected override Task InitializeAsync(CancellationToken cancellationToken, IPr
         {
             AddService(typeof(IGitHubServiceProvider), CreateService, true);
             AddService(typeof(IUsageTracker), CreateService, true);
+            AddService(typeof(ILoginManager), CreateService, true);
```

Comment:
The services that need to be initialized here are things that we need to use at VS startup. MEF is too slow at loading its catalog and causes a performance hit that we can't afford at this stage. The login manager is not one of these services, so it should definitely be an export.